sklearn/linear_model/_ridge.py

```diff
@@ -2106,6 +2110,26 @@ def fit(self, X, y, sample_weight=None):
 
 
 class _BaseRidgeCV(LinearModel):
+    _parameter_constraints = {
+        "alpha": [np.ndarray, (0.1, 1.0, 10.0)],
```

Comment:
Here you can use the `"array-like"` option:

```python
"alpha": ["array-like"]
```

Reply:
Couple additional comments:
1. The variable is `alphas` rather than `alpha`
2. We allow for a single scalar value, e.g. `alphas=1` is valid
```suggestion
        "alphas": ["array-like", Interval(Real, 0, None, closed="neither")],
```